ini/features.ini

```diff
@@ -238,7 +238,7 @@ HAS_SERVOS                             = src_filter=+<src/module/servo.cpp> +<sr
 MORGAN_SCARA                           = src_filter=+<src/gcode/scara>
 HAS_MICROSTEPS                         = src_filter=+<src/gcode/control/M350_M351.cpp>
 (ESP3D_)?WIFISUPPORT                   = AsyncTCP, ESP Async WebServer
-                                         ESP3DLib=https://github.com/luc-github/ESP3DLib/archive/master.zip
+                                         ESP3DLib=https://github.com/eduard-sukharev/ESP3DLib/archive/patch-1.zip
```

Comment:
Rather than moving this library around like a Pikey caravan, we should just maintain it under the MarlinFirmware org.

Reply:
That library is maintained by @luc-github and it's being refactored to v3.
The master branch contains a working, but old state. This fix only works for that old state. In v3 there's already a fix, but different one. Once v3 gets released and tested Marlin part should be altered for that.

Reply:
I do not understand this way to do is even acceptable

Reply:
I clarify to avoid any misunderstanding of my previous comment

@ellensp can you clarify your suggestion ?
>MKS customized the code in the library ESP3DLib to use and understand sd_busy_lock
>If you wish to do this correctly, you need to fork the ESP3DLib project, make the same changes as MKS made to it.
>Update marlin to use your version of the ESP3DLib then update marlin/Marlin/src/HAL/ESP32/u8g_esp32_spi.cpp to use sd_busy_lock.

Is that normal to kick out an existing live library (you did PR to it) to put a fork of it ?

@thinkyhead  Why esp3dLib should be under under the MarlinFirmware org ? Does every libraries used by Marlin are under MarlinFirmware org? Can you explain please ?

Reply:
@luc-github I propose merging my patch to your `master` branch. It looks like it's a proper way to go — the master branch fits current Marlin version and needs this bugfix. Once v3 gets released that solution would be dropped and substituted.

Reply:
@eduard-sukharev  I already told you I was ok for PR since the begining
But this is not what was suggested above, and not what you did in current PR

Current situation confuse me a lot about Marlin working way and I think it is legit to ask some clarifications

Reply:
> @thinkyhead Why esp3dLib should be under under the MarlinFirmware org ? Does every libraries used by Marlin are under MarlinFirmware org? Can you explain please ?

If you're the primary author and maintainer of the library then we'll point to yours. What I don't want to do is point to some other fork. It would be better if it was a separate branch under your own fork, at least.

Reply:
Ok so now I know I am no more the maintainer of esp3dlib -  I am impressed by the way things are managed in this repository

Reply:
Sorry, I misinterpreted a comment above to say that the other library was needed for the patch, but then a V3 was coming to replace that. We can easily revert this mistaken change.